src/Supervisor.ts

```diff
@@ -129,8 +130,14 @@ class Supervisor {
 
       // connect ssh & run
       try {
-        if (!ssh.connection.isConnected())
-          await ssh.connection.connect(ssh.config);
+        var ct = 1;
```

Comment:
Can we remove this exponential backoff stuff from the PR and put it in another branch/PR?